i18n/de/i18n.json

```diff
@@ -0,0 +1,106 @@
+{
+  "index": {
+    "title": "Lizenzhinweisgenerator",
+    "description": "Lizenzhinweise für Bilder aus Wikipedia und Wikimedia Commons",
+    "no-script": "Sie müssen JavaScript zur Nutzung dieser Website aktiviert.",
```

Comment:
Haha, close enough though. Something like "Zur Nutzung dieser Webseite muss JavaScript aktiviert sein." would be better.